plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2698,8 +2698,21 @@ elementgui.structuregen.projection=Projection to the terrain:
 elementgui.structuregen.separation_spacing=Structure chunk distribution:
 elementgui.structuregen.separation=Separation:
 elementgui.structuregen.spacing=Spacing:
+elementgui.structuregen.jigsaw_size=Jigsaw generation depth:
+elementgui.structuregen.jigsaw_max_distance_from_center=Maximum distance from main structure:
+elementgui.structuregen.jigsaw_pools=Jigsaw pools list
+elementgui.structuregen.jigsaw_add_pool=Add jigsaw pool
+elementgui.structuregen.jigsaw_add_pool.message=Enter the name of the custom property to be added:
```

Comment:
This message could probably be more clear and not abstract with a term "custom property"

Reply:
Property... I am blind 🤦‍♂️

Reply:
> Are there complications if we allow pool name changes (editable text field) and omit dialog? Contrary to item properties, there is no state editor to update

Reply:
Will do tomorrow.

Reply:
✔️ Fixed